proto-public/pbauth/v2beta1/traffic_permissions.proto

```diff
@@ -101,7 +101,7 @@ message DestinationRule {
   // Methods is the list of HTTP methods. If no methods are specified,
   // this rule will apply to all methods.
   repeated string methods = 4;
-  DestinationRuleHeader header = 5;
+  repeated DestinationRuleHeader headers = 5;
```

Comment:
In the RFC, this is `header` rather than `headers`. I believe we talked about the difference, but I can't remember. Could you remind me about why you made this change?

Reply:
because having a list of headers together in the same place seemed like better UX, rather than separating them up a level